src/ixdat/techniques/ec_ms.py

```diff
@@ -133,18 +134,20 @@ def ecms_calibration_curve(
                 Defaults to a new axis.
             axes_measurement (list of Axes): The EC-MS plot axes to highlight the
                 ms_calibration on. Defaults to None.
+            return_axes (bool): Whether to return the axes together with the MSCalResult.
+                Defaults to False.
```

Comment:
Should this mention the three-part return value option, where setting this to True will return 2 objects, but three if axis_measurement is also given?

Reply:
Actually, there is no need to return axes_measurement, since the user of this function must already have them. So the function only has one or two returns based on the value of `return_axes`.

Reply:
I still don't think this does what the docstring says it should. As of now the docstring says:
```text
            ax (Axis): The axis on which to plot the ms_calibration curve result.
                Defaults to a new axis.
           ...
           return_axes (bool): Whether to return the axis on which the calibration is
                plotted together with the MSCalResult. Defaults to False.
        Return MSCalResult(, Axis): The result of the ms_calibration (and calibration
            curve axis if requested)
```
and the code says:
```python
        if ax and return_axes:
            if axes_measurement:
                return cal, ax, axes_measurement
            return cal, ax
```
but, there is no reason to test for ax, it will always be there. That is also the case further up. If this is meant as a way to allow toggling the plotting of the fit on/off, then the `ax` argument should probably default to None and the "new" option documented. But as it is now, the only way that ex will ever not be True is if the user explicitly provides an argument that evals to False, which since that option isn't documented, seems fairly exotic.

If, what you meant with the `ax` option merely was, that the user can choose **where** the fit is plotted, not if it is, then both tests in this method for `if ax` can be removed.

But back to the return values which was what we were originally talking about. There still exists the 3 value return, suggest it should be:
```python
if return_axis:
    return cal, ax
return ax
```
or if you want to allow for and document not having an ax:
```python
if return_axis and ax:
    return cal, ax
return ax
```

but I suspect that the first option is really what you want.